test/remediation.test.ts

```diff
@@ -0,0 +1,68 @@
+import { test } from 'tap';
+import fs = require('fs');
+
+import plugin = require('../lib');
+import { chdirWorkspaces } from './test-utils';
+const testUtils = require('./test-utils') as any;
+
+test('remediation with constraints.txt', async (t) => {
+  chdirWorkspaces('pip-app');
+  t.teardown(testUtils.activateVirtualenv('pip-app'));
+  const upgrades = {
+    'Django@1.6.1': { upgradeTo: 'Django@2.0.1' },
```

Comment:
what if you pass in `Django@1.6.1` instead of `django@1.6.1`? Can we have a test where the cases mismatch because I know this is valid in Python

Reply:
Added, seems to be working fine.

Reply:
Added a test in upper case, still works.